src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -721,8 +721,10 @@ static ucs_status_t uct_ud_mlx5_iface_create_qp(uct_ib_iface_t *ib_iface,
     uct_ib_mlx5_qp_attr_t attr = {};
     ucs_status_t status;
 
-    attr.super     = *ib_attr;
-    attr.mmio_mode = UCT_IB_MLX5_MMIO_MODE_LAST;
+    ib_attr->cap.max_send_sge    = 2; /* ctrl + data segments */
```

Comment:
i guess you meant `UD header/s + payload`, but 1 sge is enough, as driver will create bigger tx queue than needed anyway